pcsx2/GS/GSCapture.cpp

```diff
@@ -122,6 +123,8 @@ extern "C" {
 
 namespace GSCapture
 {
+	Common::Timer s_capture_start_time;
```

Comment:
Can you move this down to where the rest of the variables are? (line 163 or so). Constants are at the beginning.

Also, should be static.